### PR TITLE
Hisat2 version update nf-core/rnaseq#870

### PR DESCRIPTION
**Pull request by RaqManzano**
Updating hisat2 to latest available version, there are some conflicht between hisat2 module and align as described here: https://github.com/nf-core/rnaseq/pull/870

### CODE REVIEW

File: combinations/hash.tsv [hunk 1]
```diff
@@ -13,8 +13,8 @@ bwa-mem2=2.2.1,samtools=1.12	bgruening/busybox-bash:0.1	0
 bwa-mem2=2.2.1,samtools=1.14	bgruening/busybox-bash:0.1	0
 bwa-mem2=2.2.1,samtools=1.15	bgruening/busybox-bash:0.1	0
 bwa-mem2=2.2.1,samtools=1.15.1	bgruening/busybox-bash:0.1	0
-hisat2=2.2.0,samtools=1.10	bgruening/busybox-bash:0.1	0
-hisat2=2.2.0,samtools=1.15.1	bgruening/busybox-bash:0.1	0
+hisat2=2.2.1,samtools=1.10	bgruening/busybox-bash:0.1	0
```
Comment (drpatelh) [inline combinations/hash.tsv:16]:
```suggestion
```

Comment (drpatelh) [reply combinations/hash.tsv:16]: We should be fine with just a container for the latest version of `samtools` right @RaqManzano ?

Comment (RaqManzano) [reply combinations/hash.tsv:16]: Absolutely!

File: combinations/hash.tsv [hunk 2]
```diff
@@ -13,8 +13,7 @@ bwa-mem2=2.2.1,samtools=1.12	bgruening/busybox-bash:0.1	0
 bwa-mem2=2.2.1,samtools=1.14	bgruening/busybox-bash:0.1	0
 bwa-mem2=2.2.1,samtools=1.15	bgruening/busybox-bash:0.1	0
 bwa-mem2=2.2.1,samtools=1.15.1	bgruening/busybox-bash:0.1	0
-hisat2=2.2.0,samtools=1.10	bgruening/busybox-bash:0.1	0
-hisat2=2.2.0,samtools=1.15.1	bgruening/busybox-bash:0.1	0
```
Comment (drpatelh) [inline combinations/hash.tsv:17]: We need to keep these lines for old containers here @RaqManzano and just append any new container entries.

Comment (RaqManzano) [reply combinations/hash.tsv]: Will put back those lines then, thanks

**Review by RaqManzano — COMMENTED** [review]
Done https://github.com/BioContainers/multi-package-containers/pull/2316/commits/8d5ae6b72683ee427fa1caece5adf4385d7dd4e1

File: combinations/hash.tsv [hunk 3]
```diff
@@ -13,6 +13,8 @@ bwa-mem2=2.2.1,samtools=1.12	bgruening/busybox-bash:0.1	0
 bwa-mem2=2.2.1,samtools=1.14	bgruening/busybox-bash:0.1	0
 bwa-mem2=2.2.1,samtools=1.15	bgruening/busybox-bash:0.1	0
 bwa-mem2=2.2.1,samtools=1.15.1	bgruening/busybox-bash:0.1	0
+hisat2=2.2.0,samtools=1.10	bgruening/busybox-bash:0.1	0
```
Comment (bgruening) [inline combinations/hash.tsv:16]: Are really both versions needed?

Comment (RaqManzano) [reply combinations/hash.tsv:16]: Apparently is necessary for old containers, maybe @drpatelh can explain better.

Comment (drpatelh) [reply combinations/hash.tsv:16]: The changes on this PR should only add a single container to overcome [this](https://github.com/nf-core/rnaseq/pull/870#issue-1383647872) issue.    This is why I suggested the previous changes.

**Review by bgruening — APPROVED** [review]

**Comment by drpatelh** [comment]
Thanks @bgruening !

**Pull request merged**